README.md

```diff
@@ -144,10 +141,17 @@ Please run the command:
 ```shell
 spectral lint step5.yaml --verbose --ruleset rulesets/semver-error.yaml; echo "exit status:" $? 
 ```
+In return:
+```shell
+Found 53 rules (42 enabled)
+Linting /Users/agentle/src/DEVWKS-2525/step5.yaml
+No results with a severity of 'error' found!
+exit status: 0
+```
 
 Perfect, the issue is now fixed!
 
-> Note that there are now 53 rules being considered and 42 are  applicable for our OpenAPI document in version 3.0.
+> Note that there are now 53 rules being considered and 42 are applicable for our OpenAPI document in version 3.0 of the OpenAPI specification.
```

Comment:
I believe you're referencing the OAS version value; please double-check that I interpreted correctly.

Reply:
yes integrated